server/routes/offers.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import { getAllCycles } from '../data';
+import { ApplicationObject, CycleObject } from '../typings';
+
+const router = express.Router();
+
+// GET /offers
+router.get('/', async (req, res) => {
+    // Gets current cycle
+    try {
+        let cycles: CycleObject[] = await getAllCycles(req.session.user._id);
+        let offers: Array<ApplicationObject> = [];
```

Comment:
I believe these two are the same but we might want to stick to one way to define arrays for consistency

Reply:
Agree, but not blocking his merge